### ISSUE TITLE
Converting RESIZE_NEAREST_NEIGHBOR

### ISSUE THREAD

**Issue by JakubGal** [tensorflow/tensorflow#41085]
**System information**
- OS Ubuntu 20.04
- TensorFlow installed from binary
- TensorFlow version (2.1,2.2,2.3)-Explained below

**Command used to run the converter or code if you’re using the Python API**
import tensorflow as tf
import numpy as np
import sys

if len(sys.argv) < 2:
    print(f"Usage: %s model.h5" % (sys.argv[0]))
    sys.exit(1)

model = tf.keras.models.load_model(sys.argv[1])
converter = tf.lite.TFLiteConverter.from_keras_model(model)
converter.optimizations = [tf.lite.Optimize.DEFAULT]
converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS_INT8]
converter.inference_input_type = tf.uint8
converter.inference_output_type = tf.uint8
def representative_dataset_gen():
  for _ in range(250):
    data = np.random.uniform(0.0, 1.0, size=(1, 416, 416, 3)).astype(np.float32)
    yield [data]
converter.representative_dataset = representative_dataset_gen
tflite_model = converter.convert()

with open('out.tflite', "wb") as f:
    f.write(tflite_model)`

**The output from the converter invocation**
error: failed while converting: 'main'
Ops that can be supported by the flex runtime (enabled via setting the -emit-select-tf-ops flag): ResizeNearestNeighbor

**h5 model**
https://drive.google.com/file/d/1iYGntXy-uAiIwxhlfOQEopqNacCPM5_P/view?usp=sharing
**Explanation of the whole problem**
https://github.com/google-coral/edgetpu/issues/152
**In short**
I tried to run the model on Coral TPU and I ran into a problem when I converted the model to .tflite using tf 2.2 and 2.3 because when I went to convert this .tflite model using edgetpu_compiler I got an error:
Edge TPU Compiler version 2.1.302470888
ERROR: Didn't find op for builtin opcode 'RESIZE_NEAREST_NEIGHBOR' version '3'

ERROR: Registration failed.

Invalid model: yolo_rcj.tflite
Model could not be parsed.

But when I used older versions of tf (1.15,2.0) I got this error:
error: failed while converting: 'main'
Ops that can be supported by the flex runtime (enabled via setting the -emit-select-tf-ops flag): ResizeNearestNeighbor
But when I used TensorFlow 2.1 to convert, it worked. 

I am writing this problem because I think it will be a problem in the future

**Comment by amahendrakar** [comment]
Was able to reproduce the issue with [TF v2.2](https://colab.research.google.com/gist/amahendrakar/26690e600826a6183b4b9e27729630f1/41085-2-2.ipynb). Facing an error stating `Ops that can be supported by the flex runtime (enabled via setting the -emit-select-tf-ops flag): ResizeNearestNeighbor.`, on running the code.

However, with [TF v2.1](https://colab.research.google.com/gist/amahendrakar/7c2126c63ee2c7b9d6ab4e263c8fad7b/41085-2-1.ipynb), [TF v2.3.0-rc0](https://colab.research.google.com/gist/amahendrakar/c96b4b320610765544e36d02ce55e997/41085-2-3.ipynb#scrollTo=Hzhl1Xj4qf-5) and [TF-nightly](https://colab.research.google.com/gist/amahendrakar/55b0bfecd2df72a1d6aa34f2c7ee3e5e/41085-tf-nightly.ipynb) I was able to convert the model successfully. Please find the attached gist. Thanks!

**Comment by Namburger** [comment]
@amahendrakar I guess this won't be fixed for `tf2.2` then, since future version will be working?

**Comment by jvishnuvardhan** [comment]
@JakubGal I ran your code and the converted model has `int32` dtype for bias (FusedBatchNormV3) in the first `Conv2D`.  Please check the attached image. Can you please share the code used for model building? Thanks!
<img width="654" alt="Screen Shot 2020-07-08 at 5 29 58 PM" src="https://user-images.githubusercontent.com/46058173/86983714-74830500-c141-11ea-9332-344462706da2.png">

**Comment by JakubGal** [comment]
@jvishnuvardhan 
First I trained the model using [darknet](https://github.com/AlexeyAB/darknet):
https://drive.google.com/drive/folders/1dm_q5hhxIC7NtbN31IepR0thMxFMytyW?usp=sharing
Then I converted the model to .h5:
https://drive.google.com/drive/folders/1s1N8PoJojF6dCY6AJnDGHWtLdZgnsYUu?usp=sharing
Next to .tflite:
https://drive.google.com/drive/folders/1nJfaqerZ97CR19TxZe2tioD0LCYzr6HN?usp=sharing
And finally I used [edgetpu_compiler](https://coral.ai/docs/edgetpu/compiler/) to compile model for Coral TPU:
https://drive.google.com/drive/folders/1MDLHuVRw5HHUpOt2_0CtXwaEwlZu7mtx?usp=sharing

And I found out another issue:
https://drive.google.com/file/d/1WPB8-rag9pDKELnMiGl2yngtM-JvKlnE/view?usp=sharing
I don't know why the QUANTIZE can't by converted. It is not a big deal but I Am wonder why.

I found a huge reduction in accuracy when using post-training quantization so I started to reimplement the model directly using Tensorflow so that I could use quantization-aware training. This might also help with this problem.But I still don't know where the mistake might be. Any ideas?

**Comment by sushreebarsa** [comment]
@JakubGal Could you please refer to [this](https://github.com/google-coral/edgetpu/issues/206) if there is any specific residual issue with RESIZE_NEAREST on edge_tpu and let us know if it helps ? Thank you!
